package tn.esprit.tic.ssa.autoloc.repository;

import org.springframework.data.jpa.repository.JpaRepository;
import tn.esprit.tic.ssa.autoloc.domain.Equipement;

public interface EquipementRepository extends JpaRepository<Equipement,Long> {
}
